docs/examples/custom_operations/custom_operator/naive_histogram/README.md

```diff
@@ -0,0 +1,48 @@
+# DALI custom operator example
+
+This is an example/template of a DALI Custom operator.
+The implemented operation is a (very) naive histogram on a grayscale image.
+The NaiveHistogram operator is a GPU-only operator, which uses a badly written
+CUDA kernel to calculate the histogram. The purpose here is not to show-case
+how the write CUDA kernels, but to present how to use CUDA kernels within
+DALI operators.
+
+# Running the example
+
+To run the example, please follow the steps:
+
+1. Clone or download the repo
+1. Install DALI wheel, for example using [these instructions](https://docs.nvidia.com/deeplearning/dali/main-user-guide/docs/installation.html#pip-official-releases)
+1. Build the `libnaivehistogram.so`:
+```bash
+$ cd naive_histogram
+$ mkdir build
+$ cd build
+$ cmake ..
+$ make -j
+```
+
+4. **(Optional)** To run the example, you need to provide some test files (actually, images). It can 
```

Comment:
I am not sure if this is optional step? Either download the DALI_extra correctly, or adjust the data paths used in test to point to directory containing user's images.

Reply:
Removed the "optional"